lms/djangoapps/program_enrollments/api/v1/tests/test_views.py

```diff
@@ -0,0 +1,148 @@
+"""
+Unit tests for ProgramEnrollment views.
+"""
+from __future__ import unicode_literals
+
+import mock
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from lms.djangoapps.courseware.tests.factories import GlobalStaffFactory
+from lms.djangoapps.program_enrollments.models import ProgramEnrollment
+from student.tests.factories import UserFactory
+
+from .factories import ProgramEnrollmentFactory
+
+
+class ProgramEnrollmentListTest(APITestCase):
+    """
+    Tests for GET calls to the Program Enrollments API.
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(ProgramEnrollmentListTest, cls).setUpClass()
+        cls.program_uuid = '00000000-1111-2222-3333-444444444444'
```

Comment:
because there are only so many UUIDs left in the world!  Also, because it's now a predictable value, which I've found is generally helpful for testing.

Reply:
I don't wanna focus on this, so I am prefixing my comment below by saying it's is super NIT.
Running out UUID is a problem on the edge of the universe, we would all be gone when that happens.
If you call UUID.uuid4() in `setupClass`, the value won't change across all the tests in this class. It means the value is predictable within the scope of this class. Isn't that enough?